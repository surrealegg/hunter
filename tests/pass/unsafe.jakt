extern function printf(anon format: raw c_char) -> usize

function main() {
    unsafe {
        printf("For now it does nothing :P\n");
    }
    unsafe {
        unsafe {
            printf("Warning check\n");
        }   
    }
}