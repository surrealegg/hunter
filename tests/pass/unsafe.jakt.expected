For now it does nothing :P
Warning check
